packages/blade/src/_helpers/storybook/Sandbox/Sandbox.native.tsx

```diff
@@ -0,0 +1,10 @@
+import dedent from 'dedent';
+import type { SandboxProps } from './Sandbox.web';
+import { Text } from '~components/Typography';
+
+// In React Native, the codesandbox doesn't work. So replacing it with normal text display for native
+function Sandbox({ children }: SandboxProps): JSX.Element {
+  return <Text>{dedent(children)}</Text>;
```

Comment:
can you add a screenshot of how it looks like on native? what if we replace it with `Code` instead of `Text`? or since we don't have multiline code variant then probably use BaseText here and maybe use the `code` font family instead of text so that the code looks like code?

Reply:
With `Text`
<img width="355" alt="image" src="https://user-images.githubusercontent.com/30949385/199654587-2a35081a-3e23-4e22-9c2c-1563c2bf5759.png">

With `BaseText` and `code` fontFamily
<img width="477" alt="image" src="https://user-images.githubusercontent.com/30949385/199654909-84c0b628-ce2f-423b-9076-ff25b88abc99.png">

Reply:
I think we can go with `BaseText` and `code` font 🤔 Later migrate to codeblock component when we implement it.

Anyway currently the Sandbox is only used in docs pages which are not available on React Native. So added this more from the perspective of if someone adds the sandbox in story

Reply:
Added BaseText